clients/js/src/compat.ts

```diff
@@ -425,14 +285,6 @@ function hookEthers6Call(
   };
 }
 
-function hookEthers5Send(send: Ethers5Call, cipher: Cipher): Ethers5Call {
-  return async (tx: Deferrable<TransactionRequest>, ...rest) => {
-    const data = await tx.data;
-    tx.data = cipher.encryptEncode(data);
-    return send(tx, ...rest);
-  };
-}
-
 function hookEthers6Send(send: Ethers6Call, cipher: Cipher): Ethers6Call {
```

Comment:
```suggestion
function hookEthersSend(send: EthersCall, cipher: Cipher): EthersCall {
```

Probably rename all later.